src/style.css

```diff
@@ -0,0 +1,109 @@
+@import url("./stars.css");
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  --white-color: #fff;
+  --blue-color: #00acdc;
+  --black-color: #060b35;
+  --gray-color: #616070;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 0;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+  padding-top: 40px;
+  width: 166px;
+  height: 36px;
+  color: var(--black-color);
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  padding: 4px 84px 0 16px;
+  margin: 0;
+  height: 14px;
+  color: var(--gray-color);
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  margin: 18px 0 0 2px;
+}
+
+.stars__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  width: 53px;
+  height: 16px;
+  padding: 0 0 0 11px;
+  text-align: right;
+  margin: 0;
+  color: var(--black-color);
+}
+
+.card__price {
+  display: flex;
+  font-size: 12px;
+  line-height: 18px;
+  padding-top: 22px;
+  margin: 0;
+  color: var(--gray-color);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  padding-left: 85px;
+  color: var(--black-color);
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  background: var(--blue-color);
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: var(--white-color);
+  margin: 16px;
+}
+
+.card__button:hover {
+  background: var(--white-color);
+  border: 1px solid var(--blue-color);
+  color: var(--blue-color);
+  width: 164px;
+  height: 38px;
```

Comment:
remove this. to fix issue with button size on hover - add border to `.card__button`